examples/wheel/wheel_test.py

```diff
@@ -384,7 +384,7 @@ def test_rule_expands_workspace_status_keys_in_wheel_metadata(self):
             "rules_python",
             "examples",
             "wheel",
-            "example_minimal_library_BUILD_USER_-0.1._BUILD_TIMESTAMP_-py3-none-any.whl",
+            "example_minimal_library{build_user}-0.1.{BUILD_TIMESTAMP}-py3-none-any.whl",
```

Comment:
This probably needs a separate feature flag as well, so that users can depend on the old behaviour.

Reply:
I made two feature flags, `incompatible_normalize_name` and `incompatible_normalize_version`. If both of them are `False`, you will get the old behavior here.